### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by bgregs514** [bgregs514/vscode-tm1#33] [General]
Hello Everyone!

I just wanted to give a quick heads up to those that are not already aware, as this continues to impact my ability to contribute to the project.

Due to ongoing intellectual property discussions, I am unable to contribute to this project.  The project currently remains in the same state that it existed in before the conversations began, and will remain that way until I am cleared to contribute again.

My apologies to those that have been following the project closely, as I know there was a lot of energy around this initially.  It is my sincere hope that I am able to contribute in the same capacity as I was when I first started this project.

Thanks,

Ben

**Comment by ykud** [comment]
Heya, I'll try to get to some of the simple issues this week (like that cloud connection), I was (and still am) swamped with work, but I can start setting some time aside for this now.

Y

**Reply by bgregs514** [reply]
Thank you!  No worries at all - always happy to have help! 😄 Unfortunately my hands are still tied, but I'm hoping to have some direction shortly.

Thanks again for all of your support, and I hope all is well!

Ben
